extensions/frontend-tiles/src/FrontendTiles.ts

```diff
@@ -25,6 +25,9 @@ function createMeshExportServiceQueryUrl(args: { iModelId: string, urlPrefix?: s
   if (args.enableCDN)
     url = `${url}&cdn=1`;
 
+  if (args.useIndexedDBCache)
+    url = `${url}&idb=1`;
```

Comment:
Why do you need to alter the URL based on whether or not the cache is used?

Reply:
Now you can remove `useIndexedDBCache` from the two `Args` types.

Reply:
Yeah I saw that too, just pushed that change a minute ago